src/main/java/net/sf/jabref/gui/EntryTypeList.java

```diff
@@ -127,16 +126,6 @@ public void addDefaultActionListener(ActionListener l) {
     }
 
     @Override
-    public void actionPerformed(ActionEvent e) {
-        // Default button pressed.
-        if (e.getSource() == def) {
-            def.setEnabled(false);
-        } else {
-            super.actionPerformed(e);
```

Comment:
Great!
The super call was not useless. However, the super class is also refactored so that its widgets now register lambda listeners that EntryTypeList now inherits.